### ISSUE TITLE
wrong pypi package for version 5.6.0

### ISSUE THREAD

**Issue by pedroburon** [twilio/twilio-python#334]
**Version:**
5.6.0

**API Subdomain (api/taskrouter/ip_messaging):**
packaging pypi

## Description
pypi package for version 5.6.0 was uploaded recently with some code written for version 6.x.x causing exceptions with certain imports.

```
$ diff -r twilio1 twilio2
Only in twilio2: base
Only in twilio2: compat.py
Only in twilio2: http
Only in twilio2/jwt: access_token
Only in twilio2/jwt: client
Only in twilio2/jwt: compat.py
Only in twilio2/jwt: taskrouter
Only in twilio2/jwt: validation
Only in twilio2: request_validator.py
Only in twilio2/rest: accounts
Only in twilio2/rest: api
Only in twilio2/rest: chat
Only in twilio2/rest: ip_messaging
Only in twilio2/rest: lookups
Only in twilio2/rest: monitor
Only in twilio2/rest: pricing
Only in twilio2/rest: taskrouter
Only in twilio2/rest: trunking
Only in twilio2: twiml
```

where twilio1 is the previous version of the pypi package 5.6.0

**Comment by shanemcd** [comment]
Yeah, this broke our builds last night...

**Comment by efossier** [comment]
@pedroburon @shanemcd My mistake, I'm working on uploading the correct package now.

**Comment by buenaonda** [comment]
Broke our builds too

**Comment by efossier** [comment]
@shanemcd @pedroburon I've removed the bad bdist package for 5.6.0, the remaining sdist package is the original untouched since it was released 2016-09-19. 

I believe I figured out my mistake however it wont let me reupload a bdist for that version as this is not allowed so I have re-released 5.6.0 as 5.6.1 with sdist and bdist_wheel packages. Can anyone confirm that this has fixed the issue?
